core/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/CarbonConstants.java

```diff
@@ -449,68 +449,6 @@ public static class LogEventConstants {
         public static final String RECORDED_AT = "recordedAt";
         public static final String CLIENT_COMPONENT = "clientComponent";
         public static final String CORRELATION_ID = "correlationId";
-
-        // keys for initiator details in audit logs.
-        public static final String INITIATOR_ID = "initiator.id";
-        public static final String INITIATOR_NAME = "initiator.name";
-        public static final String INITIATOR_TYPE = "initiator.type";
-
-        // keys for event details in audit logs.
-        public static final String EVENT_TYPE = "event.type";
-
-        // keys for target details in audit logs.
-        public static final String TARGET_ID = "target.id";
-        public static final String TARGET_NAME = "target.name";
-        public static final String TARGET_TYPE = "target.type";
-
-        // keys for data change details in audit logs.
-        public static final String DATA_CHANGE = "data.change";
-
-        public enum EventCatalog {
```

Comment:
Removing eventcatalog, since this needs to be moved to component level. We will use strings to reflect the actions in the component level. 

Anyhow, in future we need to have an API to get all the supported events in audit logs. For that, we can use OSGI service to register supported events from the component level.